### PR TITLE
Fairy Festival Lore

### PR DESCRIPTION
**Pull request by RedMageCole**
## About The Pull Request

This PR adds the lore from Lobotomy Corporation for the Fairy Festival abnormality.

This is just kind of me testing if I made a PR correctly so that I can make more lore for new Abnormalities (as well as adding existing ones to the game if need be), please let me know if I did okay!

## Why It's Good For The Game

Lore is fun! What if you want to read about these carnivorous fairies without having to boot up Lobotomy Corporation or look at the wiki?

## Changelog
:cl:
add: Added Fairy Festival lore.
/:cl:

### CODE REVIEW

File: code/modules/paperwork/records/lore/zayin.dm [hunk 1]
```diff
@@ -65,3 +65,23 @@
...
+	In our world, there are few outcomes that result from perfect kindness. Having fairies that live in a forest visit you just because they like you or that you've done many good deeds is only something that happens in fairy tales. Even though we know this truth, we still sometimes become deluded. Thinking that "The Fairies' Care" is happening because of all the good deeds we've done, or that the fairies are "favoring" us because there is something that makes us more special than the other employees."<br>
+	<br>
+	"The fairies are no more than carnivorous monsters, and their "protection" is their method to keep the meat fresh. When an employee enters another Containment Unit, they gobble them up in fear that they would lose their food to something else.<br>
+	However, the employees were greatly satisfied by the fairies' care, and we saw a drastic decrease in the accident rate, so we deemed it unnecessary to tell the truth to the employees."<br>
```
Comment (Kitsunemitsu) [inline code/modules/paperwork/records/lore/zayin.dm:87]: Add a newline at the bottom

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
I'll approve after this, linters cause a shitfit if you don't do this.

File: code/modules/paperwork/records/lore/zayin.dm (hunk 1 above)
Comment (RedMageCole) [reply code/modules/paperwork/records/lore/zayin.dm]: Just added, let me know if that works! Thank you so much!

File: code/modules/paperwork/records/lore/zayin.dm [hunk 2]
```diff
@@ -65,3 +65,24 @@
...
```
Comment (Kitsunemitsu) [inline code/modules/paperwork/records/lore/zayin.dm:87]: I forgot to add, you gotta put "} at the end and I didn't catch this last time.

Comment (RedMageCole) [reply code/modules/paperwork/records/lore/zayin.dm]: Got it, completely missed that! Thank you for the heads up— hopefully that should do it!

**Pull request merged**